packages/components/breadcrumb/src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
+        />
+        <title>Stencil Component Starter</title>
+    </head>
+    <body>
+        <script type="module" src="/build/breadcrumb.esm.js"></script>
+        <script nomodule src="/build/breadcrumb.js"></script>
+        <manager-breadcrumb></manager-breadcrumb>
+
+        <script>
+            const breadcrumb = document.querySelector('manager-breadcrumb');
+            breadcrumb.elements = [
+                { url: '#', active: false, name: 'Test' },
```

Comment:
is this a test file?

Reply:
This is a sort of documentation/example file where you can test your components in a standalone way. The template is generated by stencil (when you init module) and you can add your components to test them outside an app. it's what's launched and served when you `yarn start`